### PR TITLE
fix grammatical issues in what_is_accessibility

### PR DESCRIPTION
**Pull request by yashikabhargava**
### Description
fixed minor grammatical issues in what is accessibility.

### CODE REVIEW

File: files/en-us/learn/accessibility/what_is_accessibility/index.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ You might also think of accessibility as treating everyone the same, and giving
 
 Accessibility is the right thing to do. Providing accessible sites is part of the law in some countries, which can open up some significant markets that otherwise would not be able to use your services or buy your products.
 
-Building accessible sites benefits everyone:
+Building accessible sites benefit everyone:
```
Comment (Josh-Cena) [inline files/en-us/learn/accessibility/what_is_accessibility/index.md:50]: Original is correct

File: files/en-us/learn/accessibility/what_is_accessibility/index.md [hunk 2]
```diff
@@ -99,7 +99,7 @@ Cognitive impairment refers to a broad range of disabilities, from people with i
 A good foundation of accessibility for people with cognitive impairments includes:
 
 - Delivering content in more than one way, such as by text-to-speech or by video.
-- Easily understood content, such as text written using plain-language standards.
+- Easily understood the content, such as text written using plain-language standards.
```
Comment (Josh-Cena) [inline files/en-us/learn/accessibility/what_is_accessibility/index.md:102]: Original is correct. "Easily understood" is a modifier for "content".

File: files/en-us/learn/accessibility/what_is_accessibility/index.md [hunk 3]
```diff
@@ -131,7 +131,7 @@ When planning your project, factor accessibility testing into your testing regim
 
 - Is my date picker widget usable by people using screen readers?
 - If content updates dynamically, do visually impaired people know about it?
-- Are my UI buttons accessible using the keyboard and on touch interfaces?
+- Are my UI buttons accessible using the keyboard and on-touch interfaces?
```
Comment (Josh-Cena) [inline files/en-us/learn/accessibility/what_is_accessibility/index.md:134]: Original is correct. If you find it weird to join an adverbial phrase with a prepositional phrase, consider this:

```suggestion
- Are my UI buttons accessible to both keyboard and touch interface users?
```

**Review by Josh-Cena — CHANGES_REQUESTED** [review]

File: files/en-us/learn/accessibility/what_is_accessibility/index.md (hunk 2 above)
Comment (yashikabhargava) [reply files/en-us/learn/accessibility/what_is_accessibility/index.md:102]: Yes, now I understood it. Will undo the change.

File: files/en-us/learn/accessibility/what_is_accessibility/index.md (hunk 3 above)
Comment (yashikabhargava) [reply files/en-us/learn/accessibility/what_is_accessibility/index.md:134]: Thank you for the suggestion.

**Review by Josh-Cena — APPROVED** [review]
Thanks

**Pull request merged**